### PR TITLE
remove version as a configurable setting. defaults to 1.0.0

### PR DESCRIPTION
**Pull request by davidpesce**
This should be the only commit required for removing the version setting.

### CODE REVIEW

**Comment by davidpesce** [comment]
@garemoko I lumped in the fix for issue #52 because I had to upgrade the version to unset the admin config settings.

File: lib.php [hunk 1]
```diff
@@ -618,18 +618,17 @@ function tincanlaunch_settings($tincanactivityid){
...
         $expresult['tincanlaunchlrsduration'] = $activitysettings->lrsduration;
-        //TODO: remove version when it has been set as global
-        $expresult['tincanlaunchlrsversion'] = '1.0.0';
     }else{//use global lrs settings
         $result = $DB->get_records('config_plugins', array('plugin' =>'tincanlaunch'));
         foreach($result as $value){
             $expresult[$value->name] = $value->value;
         }
     }
+    $expresult['tincanlaunchlrsversion'] = '1.0.0';
```
Comment (garemoko) [inline lib.php:631]: Can you explain to me what this line of code is doing?

**Comment by garemoko** [comment]
I've done a code review and just have one question. I _think_ what this is doing is setting version as a global configuration value that's not available in anywhere the UI. If so, that's the right approach and this just needs testing before it can be merged. 

If anybody else wants to test this, please do!

**Comment by garemoko** [comment]
Merged into development branch (newly created). Will merge into master once development has been tested. Please make future PRs to the new development branch and I'll merge after discussion and code review; we can do testing altogether before merging into master. Please do continue to self test all code before making a PR.

File: lib.php (hunk 1 above)
Comment (davidpesce) [inline lib.php:631]: Yes, this is the only place where the version can be set and is no longer a configurable setting within the UI.

**Comment by davidpesce** [comment]
Per https://github.com/garemoko/moodle-mod_tincanlaunch/pull/57#issuecomment-97824642

This should be good to merge into master.

**Comment by garemoko** [comment]
This has a conflict. If you want to have a go at resolving, please do, otherwise I will get to it later.

**Pull request merged**